monasca-python/Dockerfile

```diff
@@ -3,5 +3,12 @@ ARG ALPINE_VERSION="3.6"
 
 FROM python:${PYTHON_VERSION}-alpine${ALPINE_VERSION}
 
-COPY build.sh /
+COPY build.sh stack-fix.c /
 RUN chmod +x /build.sh
+
+RUN apk add --no-cache ca-certificates tini && \
+  apk add --no-cache --virtual build-dep musl-dev linux-headers git make g++ linux-headers && \
+  gcc -shared -fPIC /stack-fix.c -o /stack-fix.so && \
+  apk del build-dep
+
+ENTRYPOINT ["LD_PRELOAD=/stack-fix.so", "/sbin/tini", "--"]
```

Comment:
Sure, I think that would work fine.

On a related note, after reading https://github.com/krallin/tini/issues/8 a bit closer I wonder if we'll end up un-fixing tini's signal handling with our startup scripts. Will need to do some testing to verify.